test/infrastructure/docker/hack/verify-golint.sh

```diff
@@ -18,34 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# shellcheck source=./test/infrastructure/docker/hack/utils.sh
-source "$(dirname "$0")/utils.sh"
+# shellcheck source=./hack/utils.sh
+source "$(git rev-parse --show-toplevel)/hack/utils.sh"
 
-# cd to the root path
-REPO_PATH=$(get_root_path)
+# lint capd code base
+cd_capd_root_path
 
-# create a temporary directory
-TMP_DIR=$(mktemp -d)
-
-# cleanup
-exitHandler() (
-  echo "Cleaning up..."
-  rm -rf "${TMP_DIR}"
-)
-trap exitHandler EXIT
-
-# pull the source code and build the binary
-cd "${TMP_DIR}"
-URL="http://github.com/golang/lint"
-echo "Cloning ${URL} in ${TMP_DIR}..."
-git clone --quiet --depth=1 "${URL}" .
-echo "Building golint..."
-export GO111MODULE=on
-go build -o ./golint/golint ./golint
-
-# run the binary
-cd "${REPO_PATH}"
-echo "Running golint..."
-git ls-files | grep "\.go$" | \
-  grep -v "\\/vendor\\/" | \
-  xargs -L1 "${TMP_DIR}/golint/golint" -set_exit_status
+make lint
```

Comment:
There is some whitespace weirdness here. (See the hover warning on the red)
When I ran `make verify` I got the following error.

```
- verify-whitespace.sh:
Verifying trailing whitespace...
Verifying new lines at end of files...
hack/verify-golint.sh: no newline at the end of file
Found whitespace errors!
```

Reply:
There indeed is a missing new-line.
Wonder why the ci didn't catch it!